moveit2.repos

```diff
@@ -7,4 +7,4 @@ repositories:
   ros2_control:
     type: git
     url: https://github.com/ros-controls/ros2_control.git
-    version: 2.12.0
```

Comment:
The released version of ros2_control in humble is 2.15. Doesn't belong in this backport, but I'd like to not have any required workspace dependencies with stable distros.